pages.pt_PT/common/aapt.md

```diff
@@ -0,0 +1,17 @@
+# aapt
+
+> Android Asset Packaging Tool.
+> Compila e empacotar os recursos de uma aplicação Android."
+> Mais informação: <https://elinux.org/Android_aapt>.
+
+- Listar os ficheiros contidos num arquivo APK:
+
+`aapt list {{path/to/app.apk}}`
```

Comment:
Thanks for your contribution, can you update the path placeholders in this page too.

Reply:
Done.